src/Program.cs

```diff
@@ -10,6 +10,7 @@
 using System.Threading.Tasks;
 using realmon.Configuration;
 using realmon.Utilities;
+using System.Linq;
 
```

Comment:
if you included Linq just to use string.Last() I would definitely not include it and use some other convenient way instead :) eg, `timerAsString[^1]`. yes we only do this once but to include a whold new namespace just to do this seems not really worth it (plus I don't want to encourage linq usage in this project in general :D)